Vagrantfile

```diff
@@ -152,6 +152,15 @@ fi
 SCRIPT
 
 
+INSTALL_PYTHON_ON_UBUNTU = <<-SCRIPT
```

Comment:
Instead of relying of the node number (which is hackish :stuck_out_tongue: ), we could check the distribution (by grepping in `/etc/*-release`?) and install Python only if it match Ubuntu?

Reply:
Do you mean we should provision this script for every VM? I think since we hold control over the desired OS in this same file, it's fine to keep the script straightforward and Ubuntu-specific.

Reply:
Then we should find a cleaner way than `> 5`

At least defined a named constant instead of using `5` :stuck_out_tongue:

Reply:
Change have been done for the Ubuntu nodes deployment (with python)